commit 6926cf32fc6eeb50a563da0c4f0775ffbf30a2bb
Author: Stefan Wahren <stefan.wahren@i2se.com>
Date:   Sat Oct 6 16:45:41 2018 +0200

    config: Enable Raspberry Pi voltage monitor
    
    This enables the Raspberry Pi voltage monitor as a replacement
    for the get_trottled sysfs approach in the firmware driver.
    
    Signed-off-by: Stefan Wahren <stefan.wahren@i2se.com>

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 1acb61e65c1e..b2651f8c611d 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -659,6 +659,7 @@ CONFIG_HWMON=m
 CONFIG_SENSORS_DS1621=m
 CONFIG_SENSORS_JC42=m
 CONFIG_SENSORS_LM75=m
+CONFIG_SENSORS_RASPBERRYPI_HWMON=m
 CONFIG_SENSORS_RPI_POE_FAN=m
 CONFIG_SENSORS_SHT21=m
 CONFIG_SENSORS_SHT3x=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index c2a284cc653b..0c44c45f1368 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -652,6 +652,7 @@ CONFIG_HWMON=m
 CONFIG_SENSORS_DS1621=m
 CONFIG_SENSORS_JC42=m
 CONFIG_SENSORS_LM75=m
+CONFIG_SENSORS_RASPBERRYPI_HWMON=m
 CONFIG_SENSORS_RPI_POE_FAN=m
 CONFIG_SENSORS_SHT21=m
 CONFIG_SENSORS_SHT3x=m
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1247_6926cf32fc6eeb50a563da0c4f0775ffbf30a2bb.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1247_6926cf32fc6eeb50a563da0c4f0775ffbf30a2bb.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
